### DISCUSSION TITLE
Adding support for systemd-journal-gatewayd

### DISCUSSION THREAD

**Discussion by nikola-milosa** [vectordotdev/vector#15357] [General]
Hello all!

I've been wandering whether you thought about adding systemd-journal-gatewayd as a source in vector? If not, can you achieve something similar using an http_client? Note that my interest has risen from a quite niche situation where i would require to have constant open connections and http_client cannot (at least as far as i've looked into the code) help me.
Thanks in advance!

**Comment by jszwedko** [comment]
Hi @nikola-milosa ! I think this would definitely fit in Vector's vision so we'd be happy to see [a feature request](https://github.com/vectordotdev/vector/issues/new?assignees=&labels=type%3A+feature&template=feature.yml) for it.
